src/librustc_typeck/check/method/suggest.rs

```diff
@@ -1392,7 +1395,7 @@ fn print_disambiguation_help(
         span,
         &format!(
             "disambiguate the {} for {}",
-            kind.suggestion_descr(),
+            kind.as_def_kind().descr(def_id),
```

Comment:
Why do we still need the `DefId`? Is this one of those things where the `DefKind` is too imprecise? cc @mark-i-m 

(not really relevant to this PR)

Reply:
@eddyb The reason `descr` takes a `def_id` is so that it can distinguish a crate from a module (since both are `DefKind::Mod`).